src/som/GraalSOMLanguage.java

```diff
@@ -106,7 +106,7 @@ public Object execute(final VirtualFrame frame) {
         } else {
           return universe.interpret(args);
         }
-      } catch (ProgramDefinitionError e) {
+      } catch (ProgramDefinitionError | FrameSlotTypeException e) {
```

Comment:
Hm, it seems strange that `FrameSlotTypeException` should ever escape `.interpret(.)`. Shouldn't this be handled where it occurs?